openlibrary/templates/recentchanges/index.html

```diff
@@ -9,7 +9,7 @@
     $for k in path.split("/"):
         $ label = d.get(k, k)
         $if loop.last:
-            $label
+            $label <br><br>
```

Comment:
```suggestion
            <div>$label</div>
```

Reply:
@jdlrobson can you review this addition and consider merging if you think it's the right direction?

Reply:
Also, let's please squash the changes pretty please :)

Reply:
![image](https://user-images.githubusercontent.com/51242628/77318107-6204cd00-6d04-11ea-87cc-40d2b71a8126.png)
This will lead to something like this

Reply:
```suggestion
           <p>
    $for k in path.split("/"):
        $ label = d.get(k, k)
        $if loop.last:
            $label 
        $else:
            $ p = p + "/" + k
            <a href="$p">$label</a> /
    </p>
```
in case we put a <p> then it will increase the margin from the sponsor a book bar which was looking odd something like this 
![image](https://user-images.githubusercontent.com/51242628/77318910-cd9b6a00-6d05-11ea-9f58-d24ecb23c669.png)


and in case we put a <div> in spite of a <p> that will make no change

Reply:
I don't think this needs an HTML change.

The element we want to add a margin to already is wrapped in an element and has a class subNav and has its own LESS file at `static/css/components/subNav.less`

Please undo the HTML change and add a CSS rule to get the desired effect.

https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#subnav